account_payment_partner/views/account_move_view.xml

```diff
@@ -40,8 +40,8 @@
                     '|',('company_id', '=', company_id),('company_id', '=', False)]
                 </attribute>
                 <attribute name="attrs">
-                    {'required': [('bank_account_required', '=', True)], 'readonly':
-                    [('state', '!=', 'draft')]}
+                    {'required': [('bank_account_required', '=', True),('move_type', 'in', ('in_invoice', 'in_refund'))],
+                    'readonly': [('state', '!=', 'draft')]}
```

Comment:
What about invisible?

Reply:
Yeah, but as it wasn't put, I also don't put it.

Reply:
I think that attribute needs some love too, keeping the standard domain on it

Reply:
I agree with Pedro, you should also put the `'invisible': [('move_type', 'not in', ('in_invoice', 'in_refund', 'in_receipt'))]` which was the standard.

Reply:
@LoisRForgeFlow  not true. see https://github.com/OCA/bank-payment/blob/11.0/account_payment_partner/views/account_invoice_view.xml#L34

I put invisible only for entries